cmd/earthly/flags.go

```diff
@@ -141,11 +141,25 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        envFileFlag,
-			EnvVars:     []string{"EARTHLY_ENV_FILE"},
-			Usage:       "Use values from this file as earthly environment variables, buildargs, or secrets",
+			EnvVars:     []string{"EARTHLY_ENV_PATH"},
+			Usage:       "Use values from this file as earthly environment variables",
 			Value:       defaultEnvFile,
 			Destination: &app.envFile,
 		},
+		&cli.StringFlag{
+			Name:        "arg-path",
+			EnvVars:     []string{"EARTHLY_ARG_PATH"},
+			Usage:       "Use values from this file as earthly buildargs",
+			Value:       defaultArgFile,
+			Destination: &app.argFile,
+		},
+		&cli.StringFlag{
+			Name:        "secret-path",
```

Comment:
unfortunately `--secret-file` is already taken:

```
-secret-file value                    A secret override, specified as <key>=<path> [$EARTHLY_SECRET_FILES]
```

I'm not sure what a good name for this is:
 - `dot-secret-file`
 - `secret-file-path`
 - `secret-path`
 - `override-secret-file`

Ideally we should keep all the `.env`, `.arg`, and `.secret` override options following the same naming scheme.

Reply:
My two copper go to `secret-file-path`.  `dot-secret-file` feels weird if they're not using a dot-file, e.g. `--dot-secret-file secrets`. `secret-path` worries me from a forward compatibility sentiment - if we add the ability to load secrets from a directory (instead of a file) or if we integrate something like vault, `secret-path` may all of a sudden be confusing. `override-secret-file` just feels like too many syllables.

My preference is pretty slight, though. I think the other options aren't bad, these are just a few reasons I like them slightly less.

Reply:
I switched to `secret-file-path`, as it has two votes now. thanks for the feedback.